awesome_cart/utils.py

```diff
@@ -214,3 +204,7 @@ def run_hotpatch(*args, **kwargs):
 	if not hasattr(get_item_details.process_args, "__patched"):
 		print(" - Patched: erpnext.stock.get_item_details.process_args -> awesome_cart.utils.erpnext_stock_get_item_details")
 		get_item_details.process_args = fn_wrap(get_item_details.process_args, erpnext_stock_get_item_details)
+
+def on_calculate_taxes_and_totals(doc):
```

Comment:
Can you confirm if it's okay if you only pass it the "doc" and not also the method. How does frappe.call make calls?

Reply:
This is OUR hook, only passes the doc. See: https://github.com/DigiThinkIT/erpnext/pull/37